test/test_psutil.py

```diff
@@ -710,6 +712,8 @@ def test_cpu_count(self):
         self.assertEqual(logical, len(psutil.cpu_times(percpu=True)))
         self.assertGreaterEqual(logical, 1)
         #
+        if "physical id" not in open("/proc/cpuinfo").read():
+            raise unittest.SkipTest("cpuinfo doesn't include physical id")
```

Comment:
I think this is gonna raise a ResourceWarning on python 3 because the file isn't closed properly. I would use with statement instead.